z3c/dependencychecker/package.py

```diff
@@ -20,6 +20,7 @@ class PackageMetadata(object):
     """
 
     def __init__(self, path):
+        logger.debug("Reading package metadata for %s...", path)
```

Comment:
I always tried to use single quotes, should we try to keep them consistent across the code? There are flake8 plugins for that as well

Reply:
That's where our styles differ. For short words, I tend to use single quotes. For sentences I **always** use double quotes. In 25% of the sentences you have a word like `can't` or `persons'` and I don't want to backslash-escape single quotes if I can just just double quotes.

I'm pretty sure the original code used double quotes :)